cql3/Cql.g

```diff
@@ -554,6 +554,10 @@ usingTimeoutClause[std::unique_ptr<cql3::attributes::raw>& attrs]
     : K_USING K_TIMEOUT to=term { attrs->timeout = std::move(to); }
     ;
 
+usingTimestampClause[std::unique_ptr<cql3::attributes::raw>& attrs]
+    : K_USING K_TIMESTAMP ts=intValue { attrs->timestamp = std::move(ts); }
+    ;
+
 /**
```

Comment:
Is this used?

Reply:
can we use something that assigns  a proper data type , not integer, here instead?

Reply:
> Is this used?

yes, it is. but I was debugging the grammar and forgot to go back to using that function. fixed

> can we use something that assigns a proper data type , not integer, here instead?

that's how timestamp is parsed in everywhere in our grammar. besides `api::timestamp_type` is `int64_t`

Reply:
now it's using expressions.

Reply:
@Jadw1 is this resolved?